jep/302/README.adoc

```diff
@@ -122,37 +125,81 @@ We need to finish up the work on the BOM to be more precise here.
 ////
 ====
 
-===== What to backup
+[[data_segregation]]
+===== Segregate configuration from binaries, variable build data, logs...
 
-In other words, what is going to contain `.gitignore`.
-As said above, this will likely be improved as we go.
+We will configure Jenkins to use non default values to better separate the more static and critically important things that must be snapshotted, like the configuration for jobs, or for Jenkins itself, from the more fluctuating data like build data, workspaces, exploded content for WAR or plugins, etc.
 
-.\.gitignore
-[source,gitignore,ffd]
-----
-!plugins
-!war
-----
+To keep things simple, we will keep having a single Docker volume, but introduce an additional level to separate things that must be snapshotted, and things that do not need to.
+Incidentally, this will help keep `.gitignore` short.
+
+Basically, instead of the now usual `/var/jenkins_home`, we will introduce two subdirectories:
 
-===== Segregate job configuration and build data
+* `/var/jenkins/home` (=`$JENKINS_HOME`) for to-be-snapshotted part,
+* and `/var/jenkins/var` for the rest.
 
 ////
-https://issues.jenkins-ci.org/browse/JENKINS-50164 though
+I'm not a 100% sure about /var/jenkins/var. We could put all those directly under /var/jenkins, but I fear it becomes a bit dirty over time.
+////
 
-This subpart especially is welcoming feedback.
+Here is what it will look like:
+
+```
+/var/jenkins/
+├── home
+│   ├── jobs
+│   │   └── the_job # configuration file only
+│   ├── nodes
+│   ├── plugins
+│   ├── secrets
+│   ├── updates
+│   ├── userContent
+│   └── users
+└── var
+    ├── logs # JENKINS-50291
+    │   └── tasks
+    ├── plugins # exploded plugins, using --pluginroot switch
+    ├── jobs # JENKINS-50164
+    │   └── the_job
+    │       ├── builds
+    │       └── workspace
+    └── war # using --webroot
+        ├── META-INF
+        ├── WEB-INF
+        ├── ...
+```
+
+This way, everything related to jobs, but is less important than its configuration itself is separated under a different `${JENKINS_HOME}/var/` root.
+
+Though we will for now back up these directories too, having this distinction baked in from the ground up could prove useful in the future if we need to make the snapshotting and rollback processes smarter and more robust.
```

Comment:
Wait, I thought we were only backing up `/var/jenkins/home`?

Reply:
yes, that's it. Reading, maybe I forgot to adjust a sentence.